registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -433,6 +435,7 @@
     "sha256:b1c03f2608aa325842f96229bfeb12dc39b26059a1634d9279afb16402eaa5b6": ["v1.11.0-alpha.0"]
     "sha256:b1867e662085f03376947fbfdd79465684bafed52a2d2ecfbefb949da45a0f59": ["v1.11.0-alpha.1"]
     "sha256:8508a3a08d5ba5786059e5dc58d61d945b537d1a13f225f3d3cdfde41e71d700": ["v1.11.0-alpha.2"]
+    "sha256:925c528bbb4a90affbeee1dff51b614ba16df826ae9c8f9d4ddd675577410bef": ["v1.11.0-beta.0"]
```

Comment:
SHA `925c528bbb4a` matches kubeadm-bootstrap-controller v1.11.0-beta.0 at the [staging repo](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/kubeadm-bootstrap-controller@sha256:925c528bbb4a90affbeee1dff51b614ba16df826ae9c8f9d4ddd675577410bef/details?inv=1&invt=Ab21Sw). 👍🏻